superset-frontend/cypress-base/cypress/integration/chart_list/card_view.test.ts

```diff
@@ -86,6 +86,7 @@ describe('chart card view', () => {
   it('should delete correctly', () => {
     // show delete modal
     cy.get('.ant-dropdown-trigger').last().trigger('mouseover');
+    cy.wait(5000);
```

Comment:
@adam-stasiak There are a couple of long waits in here... if you have advice on how to work around that (so we're not slowing down CI) it would be appreciated.

Reply:
It is PR https://github.com/apache/incubator-superset/pull/11312 for this where it was working without waits - I added more asssertions to make sure hover items are displayed. @rusackas maybe we can get code from this PR to Kamil's work? It is not too much effort I think - only one selector modified in code base and rest is only in tests.

Reply:
Thanks @adam-stasiak, I just merged that PR, so @kgabryje can rebase with it.

Reply:
Done